croncheck.php

```diff
@@ -130,24 +131,6 @@
 
 $now = userdate(time(), $format);
```

Comment:
Given that $now is  used by icinga.php I think this line should get moved over as well. $now as a var name gets used tons in lots of code so I think rename this to something which will never clash. It is important that this is the time that the script was called, and not the time that the script returned - eg if it took 30 seconds to run for whatever reason